rotkehlchen/tests/api/test_settings.py

```diff
@@ -173,7 +181,7 @@ def test_set_settings(rotkehlchen_api_server):
     for setting, raw_value in original_settings.items():
         if setting in unmodifiable_settings:
             continue
-
+        value: Any = None
```

Comment:
Can't you specify the possible values here etc.? Using `Any` in typing is discouraged as it's basically disabling it

```suggestion
        value: str | int | None = None
```